cmd/minishift/cmd/ssh.go

```diff
@@ -34,9 +34,10 @@ var sshCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
+
 		err := cluster.CreateSSHShell(api, args)
 		if err != nil {
-			atexit.ExitWithMessage(1, fmt.Sprintf("Cannot establish SSH connection to the VM: %s", err.Error()))
+			atexit.ExitWithMessage(1, fmt.Sprintf("Command failed or cannot establish SSH connection to the VM: %s", err.Error()))
```

Comment:
length of args == 0 => interactive, yes... but we do not know until inspection of the error if the connection failed, or the command failed.